day3/part1.py

```diff
@@ -0,0 +1,37 @@
+if __name__ == "__main__":
+    
+    # Read text file into numbers within an array
+    with open("day3/input.txt") as values:
+         binary_input = values.readlines()
+
+    # Why is len(binary_input[0]) = 6?
+    col = len(binary_input[0]) - 1   
+    row = len(binary_input)

```

Comment:
Let's get some better names for these variables too: row and col sound like "items" to me and not like "lengths"/"measurements" even. This "row" variable specifically I think is how you're representing the height of the 2d array so you could call it `height`